### ISSUE TITLE
Enhancement: animated cursor

### ISSUE THREAD

**Issue by bestouff** [alacritty/alacritty#3406]
Sometimes when the cursor jumps it can be very hard to follow where it lands. One very nice feature is to animate the cursor to follow it easily. The [`neovide`](https://github.com/Kethku/neovide) client implements it and makes it a breeze to follow, e.g.:  
![Jumping cursor](https://raw.githubusercontent.com/Kethku/neovide/master/assets/Railgun.gif)  
It would be nice to have this in `alacritty`.

**Comment by kchibisov** [comment]
Such effects are out of scope for alacritty. If you always lose your cursor in vim use `cursorline`/`cursorcolumn`. We’ll add blinking cursor in a future to help further with this issue.

**Comment by tbodt** [comment]
Why is this out of scope? It's not like you can implement this as a vim plugin.

**Comment by chrisduerr** [comment]
Because it's not a feature that we would like to maintain.
